Technical_Charter_v2.0.adoc

```diff
@@ -2,13 +2,17 @@
 
 > The purpose of having a charter for the Developer Relations Foundation is to help people understand its mission and scope. The DevRel Foundation Charter is a living document, allowing the community to propose changes and updates as the project evolves.
 
-Adopted June 3, 2024
+Adopted [[]]
 
 :toc:
 
 . **Mission and Scope of the Project** 
 .. The mission of the Project is to elevate the professional practice of developer relations and increase awareness of it as a driver of business value. 
 .. The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including definitions, frameworks, and the creation and codification of other artifacts that aid in the mission of the open collaboration project. 
+. **Definitions**
+.. Member: A Developer Relations Foundation Member (“Member”) is any individual who has formally registered with the Foundation, subscribes to its communications, and actively participates in its initiatives, working groups, events, or community activities. DRF Members are eligible to vote in Steering Committee elections and may stand for elected positions, subject to the eligibility requirements outlined in this Charter.
```

Comment:
Is this consistent with Linux Foundation use of the term `member` or is `participant` the correct term?

From Ana in a different [PR](https://github.com/DevRel-Foundation/wg-resource-aggregation/pull/11/files/c24b81c39dc6433244274024ed544de8c4fa05d6#r1915389998)
> LF-hosted projects typically use the word members to refer to the organizations sponsoring the project. Therefore, we are using participants in every DRF documentation instead to avoid confusion

Reply:
This needs to be changed to *participant* as *member* is only used for paid membership models in LF